### ISSUE TITLE
TAIL FIN E2 bug

### ISSUE THREAD

**Issue by Facehurt1** [FakeFishGames/Barotrauma#13262]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Flooding through exterior walls

![Barotrauma Screenshot 2024 01 04 - 21 26 01 48](https://github.com/Regalis11/Barotrauma/assets/115107617/e023be41-b63d-4c65-b6f9-5809132681be)

I have a seperate inside thick wall and outside tail fin E P1-P2-P3 piece fitted together

however it is still flooding even when i seal all the inside walls

https://youtu.be/E-1opilBWZA

Here is a video of it happening
![Barotrauma Screenshot 2024 01 04 - 21 46 36 04](https://github.com/Regalis11/Barotrauma/assets/115107617/3c7542b9-9b1d-48ef-890e-2c7ae8492883)

![Barotrauma Screenshot 2024 01 04 - 21 46 38 38](https://github.com/Regalis11/Barotrauma/assets/115107617/2ec7f97a-0a07-4015-8656-e0bd73c3729d)

basically you are unable to seal the ship from the inside even though the wall is sealed there.

### Reproduction steps

1. put the tail fin wall by your submarine
2. put a wall that seperates them both
3. blow up wall using explosion 2000 0 0 2000 command
4. unable to repair wall from inside no matter what

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.2.7.0 (Winter Update hotfix)

### Which operating system did you encounter this bug on?

Windows

**Comment by Facehurt1** [comment]
Hi here is better picture

hull is still leaking with inner wall sealed wtf [https://imgur.com/cSaHJ9j](https://steamcommunity.com/linkfilter/?u=https%3A%2F%2Fimgur.com%2FcSaHJ9j)

**Comment by Regalis11** [comment]
Seems to be the same issue that was reported in [[Unstable] Typhon 2 unreachable leak](https://github.com/FakeFishGames/Barotrauma/issues/13267) and should now be fixed in our private repo. :)
